src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -451,6 +452,13 @@ extension SecureStore: PrivacyPreservingProviding {
 		get { kvStore["keySubmissionMetadata"] as KeySubmissionMetadata? ?? nil }
 		set {
 			kvStore["keySubmissionMetadata"] = newValue
```

Comment:
here ist now the `analyticsSubmitter?.triggerSubmitData()` missing?

Reply:
Yeah was removed by accident with the conflicts, i pushed again